### ISSUE TITLE
Export with “Save Report Print Settings” fails

### ISSUE THREAD

**Issue by effjot** [joyfullservice/msaccess-vcs-integration#63]
I’m using version 3.1.62 with a database including a few reports. (I’ve tried both MDB and ACCDB.)

TLDR: I suspect saving the report print settings forgets to create the `reports` directory.

(It is no real problem for me, because I don’t have to save the print settings.)

Here’s what happened:

When exporting all source from a database the first time, with the “Save Report Print Settings”, I get “Runtime error 3004: Could not write to file”.

The debugger drops me in line 93 of modImportExport:

	' Check for fast save option
                If Options.UseFastSave And Not blnFullExport Then
                    If HasMoreRecentChanges(cDbObject) Then
                        Log.Increment
                        Log.Add "  " & cDbObject.Name, Options.ShowDebug
                        cDbObject.Export  ' <-- Debugger drops me here
                    Else
                        Log.Add "  (Skipping '" & cDbObject.Name & "')", Options.ShowDebug
                    End If

(Without fast save, execution stops at the `cDbObject.Export` in the else branch.)

After I have exported all source successfully once, I can export with the Save Report Print Settings option turned on (even without Fast Save).

If I use the “break in class” option, I end in line 775 of modFunctions:

	'---------------------------------------------------------------------------------------
	' Procedure : WriteBinaryFile
	' Author    : Adam Waller
	' Date      : 8/3/2020
	' Purpose   : Write binary content to a file with optional UTF-8 BOM.
	'---------------------------------------------------------------------------------------
	'
	Public Sub WriteBinaryFile(bteContent() As Byte, blnUtf8Bom As Boolean, strPath As String)

		Dim stm As ADODB.Stream
		Dim bteBOM(0 To 2) As Byte
		
		' Write to a binary file using a Stream object
		Set stm = New ADODB.Stream
		With stm
			.Type = adTypeBinary
			.Open
			If blnUtf8Bom Then
				bteBOM(0) = &HEF
				bteBOM(1) = &HBB
				bteBOM(2) = &HBF
				.Write bteBOM
			End If
			.Write bteContent
			.SaveToFile strPath, adSaveCreateOverWrite  ' <-- Debugger drops me here
		End With
		
	End Sub				

`strPath` is the first report in my DB (in alphabetical order), and at the time the execution stops, there is no directory `reports`.

After manually creating an empty directory `reports`, I can continue execution without further errors.

**Comment by joyfullservice** [comment]
Thanks for reporting the bug! This was fixed in b1731d1.
